src/retroshare/rsmsgs.h

```diff
@@ -150,13 +163,28 @@ class MsgAddress
 		explicit MsgAddress(const std::string& email, MsgAddress::AddressMode mmode)
 			: _type(MSG_ADDRESS_TYPE_EMAIL), _mode(mmode), _addr_string(email){}
```

Comment:
`MSG_ADDRESS_TYPE_EMAIL` at some point should be rename into something that doesn't make confusion with actual email ( myname@mydoman.tld )

Reply:
done